Documentation/HTKToMRTKPortingGuide.md

```diff
@@ -10,6 +10,8 @@
 | Setup                     | Place the InputManager in the scene. | Enable the input system in the Configuration Profile and specify a concrete input system type. |
 | Configuration             | Configured in the Inspector, on each individual script in the scene. | Configured via the Mixed Reality Input System Profile and its related profile, listed below. |
 
+Learn more about the [new configuration system here](MixedRealityConfigurationGuide.md).
```

Comment:
usually we're trying to avoid the "here" links (doc style guide) . It would be better if we would move those links into the text . you could link to the configuration guide in the first column on the word "Configuration" ?

Reply:
will do!